resources/stsci.edu/schemas/bounding_box-1.0.0.yaml

```diff
@@ -0,0 +1,26 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://stsci.edu/schemas/asdf/transform/bounding_box-1.0.0"
+title: >
+  Bounding box for a model.
+description: >
+  This object contains the bounding box for a model, which defines
+  the domain of valid inputs to the model. For 1D models, it is a list
+  of length 2, containing the lower bound followed by the upper bound.
+  For models with 2 or more dimensions, it is a list of lists, where the
+  list is in reverse input order and each sub-list is a list of length
+  2 containing the lower bound followed by the upper bound.
+
+oneOf:
+  - type: array
+    prefixItems:
```

Comment:
I double checked the JSON schema documentation and it looks like `prefixItems` didn't exist in draft 4.  See the draft specific note [here](https://json-schema.org/understanding-json-schema/reference/array.html#tuple-validation):

```
In Draft 4 - 2019-09, tuple validation was handled by an alternate form of the items keyword. When items was an array of schemas instead of a single schema, it behaved the way prefixItems behaves.
```